CRM/Grant/Form/GrantBase.php

```diff
@@ -377,6 +381,18 @@ public function buildCustom($id, $name, $viewOnly = FALSE, $profileContactType =
             );
             $this->_fields[$key] = $field;
           }
+          if ($field['html_type'] == 'File') {
+            $uploadFileSize = CRM_Utils_Number::formatUnitSize(ini_get('upload_max_filesize'), TRUE);
+            $uploadSize = round(($uploadFileSize / (1024 * 1024)), 2);
+            $this->addRule($field['name'], ts('%1 size exeeds %2 MB', [
```

Comment:
@monishdeb do these need to be E::ts or just ts()

Reply:
Right, E::ts would be better, thanks for pointing that, let me update that.  And there are 209 such occurrences in other places too: 
```
grep -irn " ts(" . | wc -l
     209
```
 but that can be fixed in a separate PR.

Reply:
done